src/domain/space/routing/SpaceRoutes.tsx

```
@@ -0,0 +1,130 @@
+import useUrlResolver from '@/main/routing/urlResolver/useUrlResolver';
+import { TabbedLayoutParams } from '../layout/tabbedLayout/TabbedLayoutPage';
+import SpaceAdminL0Route from '../../spaceAdmin/routing/SpaceAdminRouteL0';
+import SubspaceContextProvider from '../context/SubspaceContext';
+import { SpaceContext, SpaceContextProvider } from '../context/SpaceContext';
+import { SpacePageLayout } from '../layout/SpacePageLayout';
+import SpaceCommunityPage from '../layout/tabbedLayout/Tabs/SpaceCommunityPage/SpaceCommunityPage';
+import SpaceSubspacesPage from '../layout/tabbedLayout/Tabs/SpaceSubspacesPage';
+import SpaceKnowledgeBasePage from '../layout/tabbedLayout/Tabs/SpaceKnowledgeBase/SpaceKnowledgeBasePage';
+import SubspaceRoutes from './SubspaceRoutes';
+import useSpaceTabs from '../layout/tabbedLayout/layout/useSpaceTabs';
+import { parseInt } from 'lodash';
```

Biome (1.9.4)

[error] Do not shadow the global "parseInt" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)